drivers/dac/ad5758/ad5758.h

```diff
@@ -371,15 +374,25 @@ enum ad5758_adc_mode {
 	ADC_MODE_SINGLE_KEY_CONV,
 };
 
+enum ad5758_crc_en {
+	CRC_DIS,
+	CRC_EN
+};
+
 struct ad5758_dev {
 	/* SPI */
 	struct spi_desc *spi_desc;
 	/* GPIO */
 	struct gpio_desc *reset_n;
 	struct gpio_desc *ldac_n;
 	/* Device Settings */
-	uint8_t crc_en;
+	enum ad5758_crc_en crc_en;
```

Comment:
There is no need to create an enum for this.

Reply:
In this way, the driver's initial_param is more intuitive to config.

Reply:
If we use this approach in other drivers too, then it should be fine like this. Otherwise, I think it can be dropped, as @mchindri  suggested.

Reply:
enum ad5758_crc_en removed.